### DISCUSSION TITLE
Sqanti3 without reference annotation gtf or gff3

### DISCUSSION THREAD

**Discussion by bijendrabio** [ConesaLab/SQANTI3#299] [General] [answered]
Hello, I am working with non-model organism with newly assembled genomes. So I am wondering if its possible to run SQANTI3 for QC without annotation gtf or gff3 file.  Kindly suggest!

Regards,
B

**Comment by carolinamonzo** [comment, marked as answer]
Hi @bijendrabio, SQANTI3 needs the reference annotation of the transcriptome to be able to compare your samples to the reference and report on the quality of your long-reads transcriptome. You should try to generate an annotation of your non-model organism, without it, SQANTI3 can't be used.
